middleware/locals.js

```diff
@@ -73,6 +73,15 @@ module.exports = {
             return moment(dateString).format(format);
         };
 
+        /**
+         * View helper to represent date as relative time
+         * @param {String} dateString
+         */
+        res.locals.timeFromNow = function(dateString) {
+            moment.locale(locale);
```

Comment:
One thing I noticed when using this on the other PR – this function sets global locale, so possibly each time it's called here we change the locale of any later calls to `moment()`. You can chain it with the line below like `moment(dateString).locale(locale).fromNow()`.

Reply:
Oh, good call.